stock_picking_purchase_state/models/purchase_order.py

```diff
@@ -0,0 +1,18 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (http://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import models, fields
+
+
+class PurchaseOrder(models.Model):
+    _name = "purchase.order"
+    _inherit = ["purchase.order", "stock.picking.state.mixin"]
+    
+    picking_status = fields.Selection([
```

Comment:
Plese add here a comment referring to the Odoo known bug with mixins and translations, including the related URL